src/Toolkit/Toolkit.Maui/BasemapGallery/BasemapGallery.cs

```diff
@@ -41,9 +41,17 @@ public BasemapGallery()
         ListItemTemplate = DefaultListDataTemplate;
         GridItemTemplate = DefaultGridDataTemplate;
         ControlTemplate = DefaultControlTemplate;
-        _ = _controller.LoadFromDefaultPortal();
-    }
-
+        Loaded += BasemapGallery_Loaded;
+    }
+
+    private void BasemapGallery_Loaded(object? sender, EventArgs e)
+    {

```

Comment:
We need some code here to ensure this only triggers the first time loaded is fired (the null check isn't enough since it could return null or occur before this call finishes).